src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -181,5 +184,22 @@ private static void InitializeModulesWithConfiguration(IServiceProvider serviceP
                 service.Initialize(configuration);
             }
         }
+
+        private static void AddServerTelemetryChannelAndSamplingForFullFramework(IServiceProvider serviceProvider, TelemetryConfiguration configuration, bool DisableSampling)
```

Comment:
As per the offline discussion,we will leave this as is for now, and when there is a change in Core/channels to remove processors, we can completely ignore this boolean then.